### ISSUE TITLE
Setup issue

### ISSUE THREAD

**Issue by apio-sys** [humanmade/wp-simple-saml#30]

**Comment by mrvinceo** [comment]
Hi @apio-sys - I noticed you retracted your initial issue - I'm getting the exact same error, did you come to a resolution and if so, what was it? Many thanks, Paul

**Comment by apio-sys** [comment]
Hello. Yeah I retracted, I thought it wasn't appropriate in this place. I got help from the developper since I had a few issues. 1 I couldn't get to the /sso/metadata link (kicked back to the homepage). I got it to work by changing the permalink model to /%postname%/ and then correctly got the metadata XML.With that data I corrected the ACS on the G-Suite side which was wrong. Then I goot to the next error :  

`Uncaught Error: Call to undefined function HumanMade\\SimpleSaml\\get_site_by_path()`

I corrected that in wp-simple-saml/inc/namespace.php by adding :

`include( ABSPATH . '/wp-includes/ms-load.php' );`

after the first block and amended get_blog_id() with
`
        if ( ! is_multisite() ) {
                return 1;
        }
`
and the SSO login now works as expected. Thanks a lot to Shady for all the help provided !
